docs/source/using/troubleshooting.adoc

```diff
@@ -243,3 +243,70 @@ Workaround: Avoid installing packages or storing large files in the root filesys
 Instead, you can create a sub-directory in the *_mnt/sda1/_* persistent storage volume or define and mount xref:../using/host-folders.adoc#[host folders] that can share storage space between the host and the Minishift VM.
 
 If you want to perform development tasks inside the Minishift VM, it is recommended that you use containers, which are stored in persistent storage volumes, and reuse the xref:../using/docker-daemon.adoc#[Minishift Docker daemon].
+
+
+[[minshift-startup-check-failed]]
+== Minishift startup check failed
+
+While Minishift starts, it runs several startup checks to make sure that the Minishift VM and the OpenShift Cluster are able to start without any issues.
+If any configuration is incorrect or missing, the startup checks fail and Minishift does not start.
+
+If you want to force Minishift to start despite any failed checks, you can instruct Minishift to treat these errors as warnings as follows:
+
+- For KVM/Libvirt on Linux, run the following command:
++
+----
+$ minishift config set warn-check-kvm-driver true
+----
+
+- For xhyve on macOS, run the following command:
++
+----
+$ minishift config set warn-check-xhyve-driver true
+----
+
+- For Hyper-V on Windows, run the following command:
++
+----
+$ minishift config set warn-check-hyperv-driver true
+----
+
+The following sections describe the different startup checks.
+
+Driver plug-in configuration::
+One of the startup checks verifies that the relevant driver plug-in is configured correctly.
+If this startup check fails, review the xref:../getting-started/setting-up-driver-plugin.adoc#[setting up the driver plug-in] topic and configure the driver.
+
+Persistent storage volume configuration and usage::
```

Comment:
I'd be careful with the term "Persistent storage volume", it might be confusing with OpenShift's persistent volumes. Also we don't use this term anywhere else, so it would need some explanation. I'd try something like:

"Persistent user data partition" and then in the description, something like: "To keep state between restarts of the VM, Minishift creates a persistent partition on which configuration and user data is stored."